packages/caver-klay/caver-klay-accounts/src/index.js

```diff
@@ -110,38 +110,53 @@ Accounts.prototype.create = function create(entropy) {
 };
 
 Accounts.prototype.privateKeyToAccount = function privateKeyToAccount(privateKey, targetAddressRaw) {
-  var { privateKey: prvKey, address, isHumanReadable } = utils.parsePrivateKey(privateKey)
+  let {nonDecoupledAccount: account, klaytnWalletKeyAddress} = this.getNonDecoupledAccount(privateKey)
+
+  account.address = this.determineAddress(account, klaytnWalletKeyAddress, targetAddressRaw)
+  account.address = account.address.toLowerCase()
+  account.address = utils.addHexPrefix(account.address)
+
+  return this._addAccountFunctions(account)
+}
+
+Accounts.prototype.isDecoupled = function isDecoupled(privateKey, userInputAddress) {
```

Comment:
What about receiving an account object instead of receiving two parameters private key and address?

Reply:
This is for handling klaytn wallet key format as a privateKey parameter !

Reply:
Then, please use the first parameter name as 'KlaytnWalletKey'

Reply:
Um, first parameter can be normal privateKey and klaytnWalletKey.
In other functions also receive privateKey paremeter, and handling for both cases.

So i think privateKey or key is better than klaytnWalletKey.
How about changing to key?

Reply:
'key' is okay to me if you write some comments above this function. :)

Reply:
Okay thank you for opinion ! I renamed privateKey to key if function should handling both of them.